packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -821,14 +821,21 @@ stateMachine.grant(user, 'states:SendTaskSuccess');
 Any Step Functions state machine that has been created outside the stack can be imported
 into your CDK stack.
 
-State machines can be imported by their ARN via the `StateMachine.fromStateMachineArn()` API
+State machines can be imported by their ARN via the `StateMachine.fromStateMachineArn()` or
+by their resource name via the `StateMachine.fromStateMachineName()` API
```

Comment:
Can you add a statement clarifying that `fromStateMachineName` will only work for the same account/region as the current stack? Either that or I'm wondering if it's worthwhile to add in optional parameters for account/region so that other accounts/regions can be used. Maybe that overlaps unnecessarily with `fromStateMachineArn` though. Any thoughts on that?